packages/template-retail-react-app/app/components/_app-config/index.jsx

```diff
@@ -44,6 +44,18 @@ const AppConfig = ({children, locals = {}}) => {
 
     const appOrigin = getAppOrigin()
 
+    const proxyPaths = commerceApiConfig.proxyPath
+
+    let proxy
+    if (typeof proxyPaths === 'string') {
+        proxy = `${appOrigin}${commerceApiConfig.proxyPath}`
+    } else {
+        proxy = {
+            ...proxyPaths,
+            host: appOrigin
```

Comment:
I see that you introduced `host` as a way of using it for the base of the proxy. This probably will never happen, but if there is a proxy paths where the key is "host" then you have a naming collision. Even tho thats not going to happen is our lifetime, its best to avoid the possibility. 

You might be able to use the `reduce` function to create an object where the keys are the same, but the values are prepended by the `appOrigin`. 

Something like this...

```
...Object.keys(proxyPaths).reduce((acc, curr) => (
    ...acc,
    [curr]: `${appOrigin}/${proxyPaths[curr]}`
), {})
```